src/components/account-deletions-report.tsx

```diff
@@ -0,0 +1,84 @@
+import { useContext, useRef } from "preact/hooks";
+import { ascending, descending, flatGroup } from "d3-array";
+import * as Plot from "@observablehq/plot";
+import useRegistrationData from "../hooks/use-registration-data";
+import useElementWidth from "../hooks/use-element-width";
+import { ReportFilterContext } from "../contexts/report-filter-context";
+import { formatAsPercent, yearMonthDayFormat } from "../formats";
+import PlotComponent from "./plot";
+import Table from "./table";
+import type { ProcessedResult } from "../models/daily-registrations-report-data";
+import type { TableData } from "./table";
+import { format } from "d3-format";
+
+interface ProcessedFormattedData {
+  date: Date;
+  value: number;
+}
+
+function plot({ data, width }: { data: ProcessedFormattedData[]; width?: number }): HTMLElement {
+  return Plot.plot({
+    color: {
+      type: "ordinal",
+      scheme: "Tableau10",
+    },
+    marks: [Plot.ruleY([0]), Plot.line(data, { x: "date", y: "value" })],
```

Comment:
also unsure if it would be useful to have running average like we do in the Proofing Over Time report? With the dotted line?

https://github.com/18F/identity-reporting/blob/6ffea8ddee40ae9223f9fb975f5b089ad59278d7/src/components/proofing-over-time-report.tsx#L345-L379

Reply:
I'll give it a shot 👍

Reply:
> also unsure if it would be useful to have running average like we do in the Proofing Over Time report? With the dotted line?

Added running average line in 101af38.